docker/README.md

```diff
@@ -101,6 +118,20 @@ Unlike Docker Compose, you won’t need to set the `OP_SESSION` variable in `sci
 
 You’ll still need to set the environment variable `OP_LETSENCRYPT_DOMAIN` within `scim.env` to the URL you selected during [PREPARATION.md](/PREPARATION.md). Open that in your preferred text editor and change `OP_LETSENCRYPT_DOMAIN` to that domain name.
 
+(Optional) If you’re part of the Google Workspace provisioning beta, you will need to set up some additional secrets to use this functionality. Refer to our complete Google Workspace provisioning beta documentation for more complete information.
```

Comment:
Just a suggestion: can we put the optional sections under a collapsible heading? e.g.:

<details>
  <summary> Information for Google Workspace beta participants </summary>
  
  ## Google Workspace beta instructions
  
  First, edit the file located at `scim-examples/beta/workspace-settings.json` and enter in the appropriate details.
…
</details>

Reply:
I like this suggestion